meson.build

```diff
@@ -0,0 +1,46 @@
+project('hack-components', version: '0.0.0')
+
+install_data(
+    'hack-data.ini',
+    install_dir: join_paths(get_option('datadir'), meson.project_name())
+)
+
+install_subdir(
+    'hack-backgrounds/backgrounds',
+    install_dir: 'hack-backgrounds',
+    strip_directory: true
+)
+
+cmd = run_command('sh', '-c', 'echo $BRANCH')
+branch = cmd.stdout().strip()
```

Comment:
Now that we have meson, this (and I think the one below as well) could be passed as configuration options, no?